app/src/main/java/net/squanchy/schedule/tracksfilter/ScheduleTracksFilter.kt

```diff
@@ -28,43 +33,50 @@ class ScheduleTracksFilterActivity : AppCompatActivity() {
     private lateinit var trackAdapter: TracksFilterAdapter
 
     private var subscription: Disposable? = null
+    private var checkableTracks: List<CheckableTrack>? = null
```

Comment:
can't this be initialised with an empty list instead?

Reply:
I had it like that initially, had changed it to be nullable for consistency but I don't really mind anyway

Reply:
5c036f90337604064d5fb62377418a94c0b81bc8